providers/cloudflare/cloudflare_service.go

```diff
@@ -28,14 +28,18 @@ type CloudflareService struct {
 }
 
 func (s *CloudflareService) initializeAPI() (*cf.API, error) {
-	apiKey := os.Getenv("CLOUDFLARE_TOKEN")
+	apiKey := os.Getenv("CLOUDFLARE_API_KEY")
```

Comment:
Keys and tokens are different things in this scope.
I think it's better to rename the environment variable to better reflect what is what.